front/pages/w/[wId]/a/[aId]/execute/index.tsx

```diff
@@ -539,20 +539,27 @@ export default function ExecuteView({
     }, 0);
   };
 
-  const handleKeyPress = (event: KeyboardEvent) => {
+  const documentHandleKeyPress = (event: KeyboardEvent) => {
+    if (event.metaKey === true && event.key === "Enter" && canRun()) {
+      handleRun();
+      return;
+    }
+  };
+
+  const reactHandleKeyPress = (event: React.KeyboardEvent) => {
     if (event.metaKey === true && event.key === "Enter" && canRun()) {
       handleRun();
-      return false;
```

Comment:
IIRC this changes the behaviour -- did you test ?

Reply:
Agreed return `false` has a specific behaviour afaict